src/components/trade/Price.tsx

```diff
@@ -225,15 +229,17 @@ export const Price: React.FC<Props> = ({
     inputId: priceInverseInputId,
     precision: DEFAULT_PRECISION,
   })
-  const isDirect = priceShown === 'DIRECT'
+
+  const hideInverse = (!isPriceInverted && wasPriorityAdjusted) || (isPriceInverted && !wasPriorityAdjusted)
+  const hideDirect = (isPriceInverted && wasPriorityAdjusted) || (!isPriceInverted && !wasPriorityAdjusted)
 
   return (
     <Wrapper>
       <strong>
         Limit Price <OrderBookBtn baseToken={baseToken} quoteToken={quoteToken} />
       </strong>
       {/* using display: none to hide to avoid hook-form reregister */}
-      <PriceInputBox hidden={!isDirect}>
+      <PriceInputBox hidden={hideInverse}>
```

Comment:
This is the direct price, not the inverse, isn't it? why does it say "hide inverse"?

Reply:
yup